contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-ios.mdx

```diff
@@ -30,17 +30,41 @@ To handle this, you can use the `didReceiveFeatureFlags` notification to wait fo
 
 ```swift
 // Before SDK initialization
-NotificationCenter.default.addObserver(self, selector: #selector(receiveFeatureFlags), name: PostHogSDK.didReceiveFeatureFlags, object: nil)
-PostHogSDK.shared.setup(config)
+func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool { 
+    // your code
+
+    // Register to didReceiveFeatureFlags notification
+    NotificationCenter.default.addObserver(self, selector: #selector(receiveFeatureFlags), name: PostHogSDK.didReceiveFeatureFlags, object: nil)
+    PostHogSDK.shared.setup(config)
```

Comment:
won't compile because `config` does not exist, so the bare minimum is to create a config with the dummy dsn similar to other examples